sky/check.py

```diff
@@ -10,10 +10,20 @@
 
 
 # TODO(zhwu): add check for a single cloud to improve performance
-def check(quiet: bool = False, verbose: bool = False) -> None:
+def check(
+    quiet: bool = False,
+    verbose: bool = False,
+    user_specified_clouds: Optional[Tuple[str]] = None,
```

Comment:
Ideally yes, but then it would conflict with this import
https://github.com/skypilot-org/skypilot/blob/966ada1501ca03d745506d9dba77a2833af5b09e/sky/check.py#L7

Reply:
Yes, let's change to `from sky import clouds as sky_clouds` (like we have for `from sky import check as sky_check`). Keeps the UX clean